docs/REST_API.md

```diff
@@ -341,7 +341,7 @@ Snap task APIs provide the functionality to create, start, stop, remove, enable,
 | deadline                         | task timeout time                       |
 | creation_timestamp               | task creation time                      |
 | last_run_timestamp               | last running time of a task             |
-| hit_count                        | number of times a task ran              |
+| hit_count                        | number of times a task succeeded        |
```

Comment:
Simpler is better, so it LGTM as it is. @rashmigottipati - are you ok with keeping this as it is? If yes, please merge it. 

@mbbroberg, thank You for your contribution.